docs/source/extensions/logging.rst

```diff
@@ -14,45 +14,78 @@
 Logging
 #######
 
-Lightning supports the most popular logging frameworks (TensorBoard, Comet, etc...).
+Supported Loggers
+=================
+
+The following are loggers we support:
+
+.. note::
+    The following loggers will normally plot an additional chart (**global_step VS epoch**).
+
+.. note::
+    Depending on the loggers you use, there might be some additional charts.
```

Comment:
Can we
1) move the notes to the bottom (seems distracting especially directly in the introduction of the main section here
2) combine the two into one note (could be bullet points)

Alternatively just make it regular text (which I would prefer), because these are items that are anyway immediately visible once they open the logger.

Reply:
done!